compose/service.py

```diff
@@ -1701,33 +1701,17 @@ def rewrite_build_path(path):
     return path
 
 
-def exec_build(path, tag=None, quiet=False, fileobj=None,
-               nocache=False, rm=False, timeout=None,
-               custom_context=False, encoding=None, pull=False,
-               forcerm=False, dockerfile=None, container_limits=None,
-               decode=False, buildargs=None, gzip=False, shmsize=None,
-               labels=None, cache_from=None, target=None, network_mode=None,
-               squash=None, extra_hosts=None, platform=None, isolation=None):
+def exec_build(path, buildargs=None, cache_from=None, container_limits=None, custom_context=False,
```

Comment:
Ops... Here I think you've pushed the sorting a bit too far...
The problem here is that anyone using the method with just positional arguments would have the code broken.

Reply:
reverted